test/utils.jl

```diff
@@ -1,4 +1,4 @@
-function is_headers_equal{A<:AbstractString, B<:AbstractString}(original::AbstractArray{A}, expected::AbstractArray{B})
+function is_headers_equal(original::AbstractArray{A}, expected::AbstractArray{B}) where {A<:AbstractString, B<:AbstractString}
```

Comment:
Should probably be changed to `is_headers_equal(original::AbstractArray{<:AbstractString}, expected::AbstractArray{<:AbstractString})`

Reply:
bad bot

Reply:
Mr. Femto doesn't seem to want to change this. As this is just a style change I'll deal with this later on my own.